ddl/ddl.go

```diff
@@ -707,6 +709,7 @@ const (
 	codePrimaryCantHaveNull                    = terror.ErrCode(mysql.ErrPrimaryCantHaveNull)
 	codeWrongExprInPartitionFunc               = terror.ErrCode(mysql.ErrWrongExprInPartitionFunc)
 	codeWarnDataTruncated                      = terror.ErrCode(mysql.WarnDataTruncated)
+	codeErrTableCantHandleFt                   = terror.ErrCode(mysql.ErrTableCantHandleFt)
```

Comment:
Should add this in the ddlMySQLErrCodes  https://github.com/pingcap/tidb/pull/9821/files#diff-ae703365ad7e5ac7f61c75933fe74055R723.

Reply:
Done